index.js

```diff
@@ -7,13 +7,26 @@ const toArray = (val) => Array.isArray(val) ? val : [val].filter(Boolean);
 const copyJsRule = (config, options, ruleOptions) => {
     const { dir, config: { distDir } } = options;
 
-    const jsRuleIndex = config.module.rules.findIndex((rule) => rule.use && /\bnext-babel-loader\b/.test(rule.use.loader));
+    const jsRuleIndex = config.module.rules.findIndex((rule) => {
+        if (!rule.use) {
+            return -1;
+        }
+
+        if (typeof rule.use.loader === 'string') {
+            return /\bnext-babel-loader\b/.test(rule.use.loader);
+        }
+
+        return rule.use.findIndex((use) => /\bnext-babel-loader\b/.test(use.loader));
```

Comment:
```suggestion
        return rule.use.find((use) => /\bnext-babel-loader\b/.test(use.loader));
```

because the `findIndex` expects a boolean.